edx-olive/edx-platform
# fix: brings mo files up to speed with po files

## Description

The latest commit merged, [`0f8107b`](https://github.com/edx-olive/edx-platform/commit/0f8107b89b9fcb554c158bf15499c3af5e6c9cc7), to the branch `pooja/cherry-pick-pymongo-upgrade` resulted in 22 `.mo` file diffs.

This pull request brings the `.mo` files up-to-speed and includes the previously not-included `themes/conf/locale/en/LC_MESSAGES/django.mo` file.

## Supporting information

* **JIRA Tickets**: SE-4027, CAM2-531

## Testing instructions

* Checkout the [`0f8107b`](https://github.com/edx-olive/edx-platform/commit/0f8107b89b9fcb554c158bf15499c3af5e6c9cc7) commit
* Create a python 2.7 virtual environment and install `django` and `django-admin`. 
* Run `django-admin compilemessages`
* You should have 22 modified files and a new file
* Checkout this branch and do the same
* You should have no diffs

## Deadline

ASAP
